datawrapper/datawrapper
# Remove map marker CSS class from watermark

There was a CSS class `.marker-text` on `.watermark`. This was confusing the PDF/SVG exporter, because this class is expected to be present on map markers only.

There should be no visual change to the rendering of the watermark. This change is a prerequisite of fixing the export of watermarks.

#### Testing

1. Configure a theme to have a watermark:

    ``` json
    {
        "options": {
            "watermark": {
                "text": "DRAFT / CONFIDENTIAL"
            }
        }
    }
    ```

2. Configure any chart to use this theme.

3. See that the watermark renders well (rotated, gray, over the convent of the chart).
